nservicebus/msmq/transportconfig.md

```diff
@@ -58,29 +58,25 @@ Often when debugging MSMQ using [native tools](viewing-message-content-in-msmq.m
 
 snippet:ApplyLabelToMessages
 
+## Transactions and delivery guarantees
 
-## Controlling transaction scope options
+MSMQ transport supports all [transaction handling modes](/nservicebus/messaging/transactions.md), i.e. Transaction scope, Receive only, Sends atomic with Receive and No transactions.
 
-The following options can be configured when the MSMQ transport is working in the transaction scope mode.
+Refer to [Transport Transactions](/nservicebus/messaging/transactions.md) for detailed explanation of the supported transaction handling modes and available configuration options. 
```

Comment:
These two sentences are very similar, even linking to the same content. I would say something like

> MSMQ supports the following transaction modes:
> * Transaction scope
> * Receive only
> * Sends atomic with Receive
> * No transactions
> 
> Refer to [Transport Transactions](/nservicebus/messaging/transactions.md) for a detailed explanation of the supported transaction handling modes and available configuration options.

The fact that this is all of the transaction modes is irrelevant to the reader.